packages/autoloader/src/autoload.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+// autoload_wordpress.php generated by automattic/autoloader
+
+$class_map = require_once dirname( __FILE__ ) . '/composer/autoload_classmap_package.php';
+
+if ( ! function_exists( 'jetpack_enqueue_library' ) ) {
+
+	global $jetpack_packages;
+
+	if ( ! is_array( $jetpack_packages ) ) {
+		$jetpack_libraries = array();
+	}
+
+	function jetpack_enqueue_library( $class_name, $version, $path ) {
+		global $jetpack_packages;
+		if ( ! isset( $jetpack_packages[ $class_name ] )
+			 || version_compare( $jetpack_packages[ $class_name ] ['version'], $version, '<' )
+		) {
+			$jetpack_packages[ $class_name ] = array(
+				'version' => $version,
+				'path'    => $path,
+			);
+		}
+	}
+	// add the autoloader
+	spl_autoload_register(
+		function ( $class_name ) {
+				global $jetpack_packages;
+			if ( isset( $jetpack_packages[ $class_name ] ) ) {
+				if ( ! did_action( 'plugins_loaded' ) ) {
+					_doing_it_wrong( $class_name, 'Not all plugins have loaded yet!', '1' );
```

Comment:
Should we have the Jetpack version instead of `'1'` here?

Reply:
Yes this needs to update.

Reply:
done